crates/profiler/src/profile_builder.rs

```diff
@@ -127,13 +127,13 @@ impl ProfilerContext {
 fn build_samples(
     context: &mut ProfilerContext,
     samples: &[Sample],
-    resources_keys: &ResourcesKeys,
+    resources_keys: &ResourcesUnits,
 ) -> (Vec<pprof::ValueType>, Vec<pprof::Sample>) {
     assert!(samples
         .iter()
         .all(|x| matches!(x.sample_type, SampleType::ContractCall)));
 
-    let mut measurement_types = vec![
+    let mut sample_units = vec![
```

Comment:
Why this is not a part of ResourcesUnits?

Reply:
Can embedd it into it, this part is hardcoded